# Remote Management Request
temoto_2/RMPRequest rmp

# actions (rosrun or roslaunch):
string action

# name of the package
string package_name

# name of the node or launch file
string executable

# additional arguments
string args
---

# Remote Management Response
temoto_2/RMPResponse rmp
